apps/designer/app/designer/shared/image-manager/image-thumbnail.tsx

```diff
@@ -116,19 +109,24 @@ export const ImageThumbnail = ({
         }
       }}
       onKeyDown={(event: KeyboardEvent) => {
-        if (event.code === "Enter" && clientAsset.status === "uploaded") {
-          onChange?.(clientAsset);
+        if (event.code === "Enter" && renderableAsset.status === "uploaded") {
+          onChange?.(renderableAsset);
         }
       }}
     >
       <Thumbnail
-        css={{ backgroundImage: `url("${src}")` }}
         onClick={() => {
-          if (clientAsset.status === "uploaded") {
-            onChange?.(clientAsset);
+          if (renderableAsset.status === "uploaded") {
+            onChange?.(renderableAsset);
           }
         }}
-      />
+      >
+        <Image
+          renderableAsset={renderableAsset}
+          alt={description}
+          width={imageWidth}
```

Comment:
dunno if there is value in passing the width vs letting the image know it

Reply:
width is needed for optimisation

Reply:
actually no, you are correct with passing, since the size is defined in Thumbnail component